libevm/tooling/release/README.md

```diff
@@ -0,0 +1,26 @@
+# Release
+
+In the following, we create a release candidate tag `v1.13.14-0.2.0.rc.4`.
+
+1. Create two branches, usually from the `main` branch:
+
+    ```bash
+    git fetch origin main:main
+    git checkout main
+    git checkout -b release/v1.13.14-0.2.0.rc.4
+    git push -u origin release/v1.13.14-0.2.0.rc.4
+    git checkout -b myusername/release/v1.13.14-0.2.0.rc.4
+    git push -u origin myusername/release/v1.13.14-0.2.0.rc.4
+    ```
+
+    The `myusername/release/v1.13.14-0.2.0.rc.4` branch will be used to add "release modifications" and will target the branch `release/v1.13.14-0.2.0.rc.4` in a pull request.
+1. Run script `./cherrypick.sh` which cherry picks all Geth commits listed in [cherrypicks](cherrypicks)
+    - you may have to resolve conflicts
+1. Modify [params/version.libevm.go](/params/version.libevm.go)
+    - Change the `LibEVMVersion` to the correct final version name
```

Comment:
```suggestion
    - Change the `LibEVMVersion` to the correct final version string (`version.libevm_test.go` will ensure that this is correctly formatted)
```

And also make it the last point.

Reply:
Bullet points are meant to have no order unlike numbered points, but sure, done 👍